source/includes/stackpath/_delivery_domains.md

```diff
@@ -117,4 +117,45 @@ Delete a delivery domain of a site in a given [environment](#administration-envi
 Attributes | &nbsp;
 ------- | -----------
 `taskId` <br/>*string* | The task id related to the site deletion.
+`taskStatus` <br/>*string* | The status of the operation.
+
+<!-------------------- CREATE A DELIVERY DOMAIN -------------------->
+
+### Create a delivery domain
+
+```shell
+curl -X POST \
+    -H "MC-Api-Key: your_api_key" \
+    -d "request_body" \
+    "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/deliverydomains?siteId=a2fefb15-7e31-4c72-87e0-5a892e91c8d9&operation=Create"
```

Comment:
I don't think the `&operation=Create` is necessary at the end.

Reply:
removed 0db866c